### PR TITLE
65 create a route for job advert display

### PR DESCRIPTION
**Pull request by WojciechPtas**

### CODE REVIEW

File: controllers/jobAdvertController.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    }
+    catch (e) {
+        console.log(e);
+        res.sendStatus(500);
+    }
+}
+
+module.exports = {
+    showAdvert
+}
```
Comment (macibol775) [inline controllers/jobAdvertController.js:18]: endl

File: routes/advertRoutes.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const jobAdvertRoute = require('../controllers/jobAdvertController');
+
+const router = express.Router();
+
+router.get('/view', jobAdvertRoute.showAdvert);
+
+module.exports = router;
```
Comment (macibol775) [inline routes/advertRoutes.js:8]: endl

File: routes/advertRoutes.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const jobAdvertRoute = require('../controllers/jobAdvertController');
```
Comment (macibol775) [inline routes/advertRoutes.js:2]: oj chyba nie route

File: views/jobAdvert/show_advert.ejs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            <h5 class="col-sm-8 mx-auto">Salary: <%=jobAdvert.salary %></h5>
+            <h5 class="col-sm-8 mx-auto">Location: <%=jobAdvert.location %></h5>
+            <h5 class="col-sm-8 mx-auto">Expiry Date<%=jobAdvert.expiryDate %></h5>
+            <div class="col-sm-8 mx-auto">
+                <span class="errorMessageText" id="message"></span>
+            </div>
+        </div>
+    </div>
+</div>
+<%- include('../partials/footer.ejs') %>
```
Comment (macibol775) [inline views/jobAdvert/show_advert.ejs:16]: endl

**Review by macibol775 — CHANGES_REQUESTED** [review]
route, not controller

**Review by macibol775 — APPROVED** [review]
add ENDLINES next time

**Review by MasterCoookie — APPROVED** [review]

**Pull request merged**